src/handlebars/partials/footer.handlebars

```diff
@@ -25,16 +25,8 @@
   <div class="margin-auto" id="footer-social-bar">{{> social-bar }}</div>
 
   <p style="line-height: 1rem;">Java and OpenJDK are trademarks or registered trademarks of Oracle and/or its affiliates. Other names may be trademarks of their respective owners.</p>
-
-  <!--
-  <div style="margin-top: 1.5rem;" id="google_translate_element"></div>
-  <script type="text/javascript">
-    function googleTranslateElementInit() {
-      new google.translate.TranslateElement({pageLanguage: 'en', includedLanguages: 'en,pt,zh-CN', layout: google.translate.TranslateElement.InlineLayout.SIMPLE}, 'google_translate_element');
-    }
-  </script>
-  <script type="text/javascript" src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
-  -->
+  <br>
+  <p style="line-height: 1rem;"><strong>Spotted an issue with the website?</strong><br><a target="_blank" class="light-link" href="https://github.com/AdoptOpenJDK/openjdk-website/issues">Raise a GitHub issue</a> or <a target="_blank" class="light-link" href="https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md">contribute a change!</a></p>
```

Comment:
Not sure, looks a bit weird?

<img width="1440" alt="screen shot 2017-06-02 at 10 25 37" src="https://cloud.githubusercontent.com/assets/20062773/26719797/dcc79afe-477d-11e7-9d70-05848c886671.png">